README.md

```diff
@@ -53,6 +54,17 @@ npx simple-git-hooks
 
 ## `site` Setup
 
+### Local env vars
+
+Inside the `site` folder create a file called `.env.local` with the following content.
```

Comment:
Why are these in an env file now and not `wrangler.toml` or an uploaded cloudflare secret like the rest?

Reply:
these vars are used in the webpack.config.js, this file doesnt know about CF secrets